README.rst

```diff
@@ -80,7 +80,6 @@ software provided on the GitHub-hosted runners.
 For information about the specific versions of software the runners
 use, please see the following resources:
 
-- `ubuntu-18.04 Runner Information <https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu1804-README.md>`
 - `ubuntu-20.04 Runner Information <https://github.com/actions/virtual-environments/blob/main/images/linux/Ubuntu2004-Readme.md>`
```

Comment:
a link to the ubuntu 22.04 runner information should be added (https://github.com/actions/runner-images/blob/main/images/linux/Ubuntu2204-Readme.md)